flink-ml-parent/flink-ml-lib/src/main/java/org/apache/flink/ml/operator/common/statistics/basicstatistic/CorrelationResult.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.flink.ml.operator.common.statistics.basicstatistic;
+
+import org.apache.flink.ml.common.linalg.DenseMatrix;
+import org.apache.flink.ml.common.utils.TableUtil;
+import org.apache.flink.types.Row;
+
+import java.util.ArrayList;
+
+/**
+ * It is correlation result, which has colNames and correlation values.
+ */
+public class CorrelationResult {
```

Comment:
This class seems to be a fit for an inner class of basesummarizer/basesummary ? I do not see this class being used outside of the summary perspective. 

Althought the toString() part can be a Utility function for DenseMatrix IMO and can be factor out?